app/models/solr_document.rb

```diff
@@ -174,6 +174,18 @@ def numismatics_record?
     solr_document_id&.start_with? 'coin'
   end
 
+  def in_a_special_collection?
+    holdings_1display = self['holdings_1display']
+    return true unless holdings_1display
+
+    JSON.parse(holdings_1display)
+        &.values
+        &.any? do |holding|
+          location_code = holding['location_code']
+          Bibdata.holding_locations.dig(location_code, 'aeon_location') if location_code
```

Comment:
We could does check at the library level instead if it's desirable just leave out everything in MUDD, SC, Marquand, and EAL